js/js.js

```diff
@@ -0,0 +1,30 @@
+$(function(){
+	//listen for submit event and execute a function to change background
+	$("form").submit(changeBackgroundImage);
```

Comment:
@cliffjadoo great that you use `.submit` for this. You could also have used `.click` or `.on`